addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -7,92 +7,320 @@
-                            <li class="breadcrumb-item active" aria-current="page"><span t-field="job.name"/></li>
-                        </ol>
-                    </nav>
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <img src="/website_hr_recruitment/static/src/img/job_image_8.jpg" class="img-fluid" alt="About us"/>
+                            <h4 class="mt24 mb8">About us</h4>
+                            <p class="pb8">
+                                We are a team of passionate people whose goal is to improve everyone's life through disruptive products.
+                                We build great products to solve your business problems.
+                            </p>
+                            <div class="s_hr text-left pt16 pb16" data-snippet="s_hr" data-name="Separator">
+                                <hr class="w-100 mx-auto" style="border-top-width: 1px; border-top-style: solid; border-top-color: rgb(206, 212, 218) !important;"/>
+                            </div>
+                            <p id="connect">
+                                <h5 class="mb-3">Connect with us</h5>
+                                <ul class="list-unstyled text-nowrap">
+                                    <li><i class="fa fa-comment fa-fw mr-2"></i><span><a href="/contactus">Contact us</a></span></li>
+                                    <li><i class="fa fa-envelope fa-fw mr-2"></i><span><a href="mailto:info@yourcompany.example.com">info@yourcompany.example.com</a></span></li>
+                                    <li><i class="fa fa-phone fa-fw mr-2"></i><span class="o_force_ltr"><a href="tel:+1 (650) 691-3277">+1 (650) 691-3277</a></span></li>
+                                </ul>
+                                <!-- Social media -->
+                                <div class="s_share text-left" data-name="Social Media">
+                                    <h5 class="s_share_title d-none">Follow us</h5>
+                                    <a href="/website/social/facebook" class="s_share_facebook" target="_blank">
+                                        <i class="fa fa-facebook btn btn-outline-primary btn-lg shadow-sm" style="border-radius: 100px;"></i>
+                                    </a>
+                                    <a href="/website/social/twitter" class="s_share_twitter" target="_blank">
+                                        <i class="fa fa-twitter btn btn-outline-primary btn-lg shadow-sm" style="border-radius: 100px;"></i>
+                                    </a>
+                                    <a href="/website/social/linkedin" class="s_share_linkedin" target="_blank">
+                                        <i class="fa fa-linkedin btn btn-outline-primary btn-lg shadow-sm" style="border-radius: 100px;"></i>
+                                    </a>
+                                </div>
```

Comment:
I would replace with this:
```suggestion
                                <div class="s_social_media text-left" data-snippet="s_social_media" data-name="Social Media" contenteditable="false">
                                <h5 class="s_social_media_title d-none" contenteditable="true">Follow us</h5>
                                <a href="/website/social/facebook" class="s_social_media_facebook" target="_blank">
                                    <i class="fa fa-facebook rounded-circle shadow-sm" contenteditable="false">​</i>
                                </a>
                                <a href="/website/social/twitter" class="s_social_media_twitter" target="_blank">
                                    <i class="fa fa-twitter rounded-circle shadow-sm" contenteditable="false">​</i>
                                </a>
                                <a href="/website/social/linkedin" class="s_social_media_linkedin" target="_blank" contenteditable="true">
                                    <i class="fa fa-linkedin rounded-circle shadow-sm d-none" contenteditable="false">​</i>
                                </a>
                                <a href="/website/social/instagram" class="s_social_media_instagram d-none" target="_blank">
                                    <i class="fa fa-instagram rounded-circle shadow-sm" contenteditable="false">​</i>
                                </a>
                                <a href="/website/social/youtube" class="s_social_media_youtube d-none" target="_blank">
                                    <i class="fa fa-youtube rounded-circle shadow-sm" contenteditable="false">​</i>
                                </a>
                                <a href="/website/social/github" class="s_social_media_github d-none" target="_blank">
                                    <i class="fa fa-github rounded-circle shadow-sm" contenteditable="false">​</i>
                                </a>
                                <a href="/" class="text-800 d-none">
                                    <i class="fa fa-home rounded-circle shadow-sm" contenteditable="false">​</i>
                                </a>
                            </div>
```

Using the new `social_media` block, the user will be able to manage links to these social networks more easily. The `s_share` block should only be used for sharing on social networks.

Reply:
Indeed. I'm replacing it. Thanks!